File: csharp/two-fer/TwoFer.cs
using System;

public static class TwoFer
{
    public static string Speak() => Speak("you");

    public static string Speak(string name)
    {
        if (name.ToLower() == "Paulo".ToLower())
            return $"One for {name}, one for me";

        return $"One for {name}, one for me.";
    }
}

File: go/two-fer/two_fer.go
// Package twofer implements two for one. One for you and one for me.
package twofer

// ShareWith implements two for one. One for you and one for me.
// Given a name, return a string with the message: `One for X, one for me.`
func ShareWith(name string) string {
	if len(name) == 0 {
		name = "you"
	}

	return "One for " + name + ", one for me."
}
